apps/explorer/src/ui/Card.tsx

```diff
@@ -10,17 +10,26 @@ const cardStyles = cva(null, {
             default: 'bg-gray-40',
             highlight: 'bg-success-light',
             lightBlue: 'bg-sui/10',
+            white: 'bg-white',
         },
         rounded: {
             lg: 'rounded-lg',
             xl: 'rounded-xl',
+            '2xl': 'rounded-2xl',
         },
         spacing: {
             none: '',
             sm: 'px-5 py-4',
             md: 'p-5',
             lg: 'p-8',
         },
+        border: {
+            gray45: 'border border-gray-45',
+        },
+        shadow: {
+            none: '',
+            default: 'border-none shadow-default',
```

Comment:
Probably needs a better name given the default isn't the default

Reply:
@plam-ml  Should still do this. Either rename this or make it the default (probably just rename). I get that this is what this is called in the figma but `default` has a meaning in code and this violates that meaning

Reply:
@Jordan-Mysten so there is already a `DEFAULT` boxshadow merged on main here 
https://github.com/MystenLabs/sui/blob/main/apps/core/tailwind.config.ts#L96

I would like to use this, unless we really need it to be other name since I am not sure if @pchrysochoidis is also using it anywhere else.

PR merged with that name https://github.com/MystenLabs/sui/pull/11328